config/adminlte.php

```diff
@@ -110,15 +122,17 @@
         [
             'text' => 'Change Password',
             'url' => 'admin/settings',
-            'icon' => 'lock'
+            'icon' => 'lock',
+            //'permission'=>'no-access',
```

Comment:
The configuration here is already an example, so I would prefer to not comment this out.